rt/features/logging/src/main/java/org/apache/cxf/ext/logging/event/DefaultLogEventMapper.java

```diff
@@ -46,17 +44,25 @@
 import org.apache.cxf.ws.addressing.ContextUtils;
 
 public class DefaultLogEventMapper {
-    private static final Set<String> BINARY_CONTENT_MEDIA_TYPES;
+    private static final Set<String> DEFAULT_BINARY_CONTENT_MEDIA_TYPES;
```

Comment:
Trivial improvement suggestion: since this field seems to want to be immutable, I think it's better to initialize it like this: https://github.com/spring-projects/spring-framework/blob/master/spring-context/src/main/java/org/springframework/format/datetime/DateTimeFormatAnnotationFormatterFactory.java#L43
I know this PR just renames the field, but maybe the initialization can be improved too 🙂

Reply:
Actually, I would remove this set by default. But forcing everyone to tune it is not good.

Reply:
I think what @andrei-ivanov meant here is that this patch could use Collections.unmodifiableSet instead?

Reply:
But, i need modifiable set, in order to be able to customize it.
May be remove DEFAULT_BINARY_CONTENT_MEDIA_TYPES from class and stay only binaryContentMediaTypes varible? And initialize it in constuctor.

Reply:
Hmm, as far as I see, `DEFAULT_BINARY_CONTENT_MEDIA_TYPES` doesn't get modified, it is copied to `binaryContentMediaTypes` on line 60 and that set is modified.

Reply:
Logically, we don’t need this set at all as a separate instance.
I propose to abandon it. And configure only the editable set at once.
    public DefaultLogEventMapper() {
        binaryContentMediaTypes.add("application/octet-stream");
        binaryContentMediaTypes.add("application/pdf");
        binaryContentMediaTypes.add("image/png");
        binaryContentMediaTypes.add("image/jpeg");
        binaryContentMediaTypes.add("image/gif");
    }

What do you think? Would that be more correct?

Reply:
No I think it's easier just to leave it as it is. I'll make the change to use an unmodifiable set locally.

Reply:
Thanks. Than i close pull request.

Reply:
No, please re-open it, I will merge it and fix up the other thing locally.

Reply:
I apologize for the delay